README.md

```diff
@@ -173,6 +176,31 @@ For processing PDF files, the external command line tool [`OCRmyPDF`](https://gi
 ### Images
 For processing single images (currently `jpg` and `png` are supported), `ocrmypdf` converts the image to a PDF. The converted PDF file will then be OCR processed and saved as a new file with the original filename and the extension `.pdf` (for example `myImage.jpg` will be saved to `myImage.jpg.pdf`). The original image fill will remain untouched.
 
+## Troubleshooting
+
+### Generic troubleshooting guide
+
+Since this app does it's main work asynchronously, controlled by the NC cron, the troubleshooting gets slightly more complicated. That's why we suggest to follow this guide if you're facing any issues:
```

Comment:
```suggestion
Since this app does its main work asynchronously, controlled by the NC cron, the troubleshooting gets slightly more complicated. That's why we suggest to follow this guide if you're facing any issues:
```